src/pybind/mgr/dashboard/controllers/ui_settings.py

```diff
@@ -0,0 +1,49 @@
+import cherrypy
+from . import ApiController, RESTController,Endpoint
+from .. import logger, mgr
+from ..settings import Settings
+from ..security import Scope
+import urllib.parse as urlparse
+import json
+from ast import literal_eval
+
+@ApiController('ui-settings')
+class UiSettings(RESTController):
+    GrafanaObj = {}
```

Comment:
After adding the following I was able to set new values for grafana:
```suggestion
    RESOURCE_ID = "name"
    
    GrafanaObj = {}
```